src/OpenConext/Profile/Value/ConsentType.php

```diff
@@ -70,6 +70,14 @@ public function equals(ConsentType $other)
     }
 
     /**
+     * @return bool
+     */
+    public function isExplicit()
```

Comment:
`isExplicit` is easier to use in Twig. I added `isImplicit`, because one would expect it to be there.
If/when more ConsentTypes are added, a more scalable solution could be implemented.